src/PhotoNim/shapes.nim

```diff
@@ -49,7 +38,21 @@ type
 
 
 proc newAABox*(min = newPoint3D(0, 0, 0), max = newPoint3D(1, 1, 1), transf = Transformation.id): AABox {.inline.} = AABox(transf: transf, aabb: some(AABB((min, max))))
-proc newSphere*(transf = Transformation.id): Sphere {.inline.} = Sphere(transf: transf)    
+
+proc newSphere*(center: Point3D, radius: float32): Sphere {.inline.} = 
```

Comment:
Didn't we say that we weren't interested in this approach and that we preferred to characterize the shapes purely based on the transformation associated with them?